plugins/chat/assets/javascripts/discourse/services/chat-subscriptions-manager.js

```diff
@@ -228,6 +229,8 @@ export default class ChatSubscriptionsManager extends Service {
       channel.threadsManager
         .find(busData.channel_id, busData.thread_id)
         .then((thread) => {
+          thread.lastReply = ChatThreadLastReply.create(busData);
```

Comment:
Not quite sure if this is  the nicest place...this is how we update the last reply for the thread when a new message comes in

Reply:
Seems good to me so far